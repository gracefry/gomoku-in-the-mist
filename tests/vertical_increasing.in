place S18
place A1
place S17
place A2
place S16
place A3
place S15
place A4
place S14
