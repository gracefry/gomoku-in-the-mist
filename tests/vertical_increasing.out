Black wins!
S18A1S17A2S16A3S15A4S14
Thank you for playing!
